packages/components/src/DateTimePickers/views/MonthYearView/MonthYearView.component.js

```diff
@@ -17,7 +17,7 @@ function MonthYearView(props) {
 				icon="talend-arrow-left"
 				onClick={props.onBackClick}
 				tabIndex={props.allowFocus ? 0 : -1}
-				link
+				className="btn-tertiary btn-info"
```

Comment:
The back button is not described as a tertiary button in guideline

Reply:
From my point of view, it is a tertiary button as the pictures showed in guideline.
Will confirm with ux.

Reply:
Hi Jimmy, checked with ux, it is a tertiary button.
and also the month-year button in date picker.
I have just committed the code to add the style for it.